cmd/pbackend/statuser.go

```diff
@@ -131,7 +131,7 @@ func checkSourceAvailabilityAzure(ctx context.Context) {
 				ResourceType: "Application",
 			}
 			// TODO: check if source is avavliable - WIP
```

Comment:
Can you drop this TODO? This patch actually implements the missing functionality.